python/cugraph/mg_pagerank/mg_pagerank_wrapper.pyx

```diff
@@ -0,0 +1,31 @@
+from libc.stdint cimport uintptr_t
+from c_mg_pagerank cimport *
+import cudf
+import numpy as np
+
+def mg_pagerank(src_ptrs_info, dest_ptrs_info):
+    print("mg_pagerank.py: ", src_ptrs_info, dest_ptrs_info)
+    
+    cdef gdf_column* src_column_ptr = <gdf_column*>malloc(len(src_ptrs_info) * sizeof(gdf_column))
+    cdef gdf_column* dest_column_ptr = <gdf_column*>malloc(len(dest_ptrs_info) * sizeof(gdf_column))
+    
+    N = len(src_ptrs_info);
+    for i in range(N):
+        src_column_ptr[i] = get_gdf_column_ptr(src_ptrs_info[i]["data"][0], src_ptrs_info[i]["shape"][0])
+        dest_column_ptr[i] = get_gdf_column_ptr(dest_ptrs_info[i]["data"][0], dest_ptrs_info[i]["shape"][0])
+
+    cdef gdf_column* pr_ptr = <gdf_column*>malloc(len(src_ptrs_info) * sizeof(gdf_column))
+    gdf_dummy_multi_pagerank(<const size_t>N,
+                     <gdf_column*> src_column_ptr,
+                     <gdf_column*> dest_column_ptr,
+                     <gdf_column*> pr_ptr,
+                     <float> 0.85,#damping_factor,
+                     <int> 20 #max_iter
+                     )
+
+    data = rmm.device_array_from_ptr(<uintptr_t> pr_ptr.data,
+                                            nelem=pr_ptr.size,
+                                            dtype=np.int32)
+    x = cudf.DataFrame()
+    x['pagerank'] = cudf.Series(data)
```

Comment:
Let's add a place holder for vertex_indexes

Reply:
added